### PR TITLE
Remove User Functionality Added

### PR DESCRIPTION
**Pull request by echarlotte612**
Added delete user page which verifies email against password and then verifies password against a second entry before removing the user account from mongodb. User is then sent back to signup page.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ function App() {
             <Route exact path = "/" component = {Navbar} />
             <Route exact path = "/login" component = {Navbar} />
             <Route exact path = "/logout" component = {Navbar} />
+            <Route exact path = "/deleteUser" component = {NavbarLI} />
```
Comment (nmunro) [inline frontend/src/App.js:28]: Maybe deleteAccount?

File: frontend/src/components/deleteUser/DeleteUser.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        if (event.target.id === "verifyPassword"){setVerifyPassword(event.target.value)};
+    }
+    const sendDataToExpress = (event) => {
+        event.preventDefault();
+        axios.delete("http://localhost:5000/users", {data: {
+            email: email,
+            password: password,
+            verifyPassword: verifyPassword
+        }}).then((res) => {
+            console.log(res.data);
```
Comment (nmunro) [inline frontend/src/components/deleteUser/DeleteUser.js:24]: Probably don't need this line anymore?

File: frontend/src/components/deleteUser/DeleteUser.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            }
+        }).catch((err) => {
+            alert(`Account not deleted.`)
+        })
+    }
+    return(<div>
+            {(sessionStorage.getItem("login")) ?
+            <div ClassName ="deleteUser-background">
+                <form onSubmit={sendDataToExpress} className="deleteUserForm">
+                    <h2>😭😭😭😭😭😭😭😭<br/> Delete Game Guardians Account?</h2>
```
Comment (nmunro) [inline frontend/src/components/deleteUser/DeleteUser.js:43]: Haha, nice!

File: frontend/src/components/main/main.js [hunk 4]
```diff
@@ -18,16 +18,17 @@ function MainPage() {
                     <div className="recentGames">
                         <h1 className="recentGamesHeader">Recent Games: </h1>   
                         <Game />
-                        <Game />  
-                        <Game />  
-                        <Game />   
+                        <Game />
```
Comment (nmunro) [inline frontend/src/components/main/main.js:21]: I'm guessing you could have an express route for '/latestGames' and you can get them here and pass the data as props into the Game components?

**Review by nmunro — APPROVED** [review]

**Review by amandaforni — APPROVED** [review]
Well done!!!

File: frontend/src/components/main/main.js (hunk 4 above)
Comment (echarlotte612) [reply frontend/src/components/main/main.js:21]: Yeah I think that's the plan!

**Pull request merged**